climada/entity/exposures/crop_production.py

```diff
@@ -579,16 +618,16 @@ def init_full_exp_set_isimip(input_dir=None, filename=None, hist_mean_dir=None,
     if not input_dir:
         input_dir = INPUT_DIR
     if not hist_mean_dir:
-        hist_mean_dir =HIST_MEAN_PATH
+        hist_mean_dir = HIST_MEAN_PATH
     if not output_dir:
         output_dir = OUTPUT_DIR
     if yearrange is None:
         yearrange = YEARCHUNKS[isimip_version]['histsoc']['yearrange']
     if not unit:
         unit = 't/y'
 
-    filenames = [f for f in Path(hist_mean_dir).iterdir()
-                 if f.is_file and not f.name.startswith('.')]
+    filenames = [f for f in listdir(hist_mean_dir) if (isfile(join(hist_mean_dir, f))) if not
+                 f.startswith('.')]
```

Comment:
Why did you decide to remove the use of `pathlib`? This is the new recommended convention in the CLIMADA context.